rustfmt.toml

```diff
@@ -0,0 +1,55 @@
+indent_style = "Block"
+use_small_heuristics = true
+binop_separator = "Front"
+combine_control_expr = true
+comment_width = 80
```

Comment:
This could be made large, clippy uses 100.

Reply:
does it make a difference, given that comments are not wrapped? is it just for warnings?